docs/reference/builder.md

```diff
@@ -738,6 +738,11 @@ To view an image's labels, use the `docker inspect` command.
         "other": "value3"
     },
 
+> **Note**:
+> When providing a `LABEL` in a Dockerfile, be sure to use double quotes and not single quotes. Particularly when you are 
+> using string interpolation (e.g. `LABEL example="$env_var") single quotes will take the string as is without
```

Comment:
wondering though; why put quotes there in the first place? ie; this would work as well;

    LABEL example=$ENV_VAR